app/src/main/java/org/jellyfin/androidtv/ui/AlphaPicker.kt

```diff
@@ -0,0 +1,40 @@
+package org.jellyfin.androidtv.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.Button
+import android.widget.LinearLayout
+import androidx.core.view.children
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.databinding.AlphaPickerButtonBinding
+
+class AlphaPicker(
+	context: Context,
+	attrs: AttributeSet? = null
+) : LinearLayout(context, attrs) {
+	var onAlphaSelected: (letter: Char) -> Unit = {}
+
+	init {
+		val letters = "#${resources.getString(R.string.byletter_letters)}"
+		letters.forEach { letter ->
+			// NOTE: We must use a Button here and NOT AppCompatButton due to focus issues on Fire OS
```

Comment:
I don't this comment matters now :p

Reply:
Well it does, but maybe it should be moved to the XML file?

Reply:
Moved it to the XML file.